run.py

```diff
@@ -167,8 +167,8 @@ def main():
             if ulimit.startswith('nofile='):
                 ulimit_nofile = True
             docker_args += ["--ulimit", ulimit]
-    if ulimit_nofile == False:
-        docker_args += ["--ulimit", 'nofile='+str(DEFAULT_ULIMIT_NOFILE)]
+    if not ulimit_nofile:
+        docker_args += ["--ulimit", 'nofile=' + str(DEFAULT_ULIMIT_NOFILE)]
```

Comment:
This would probably be more pythonic and consistent with other strings in the file:
```suggestion
        docker_args += ["--ulimit", "nofile=%s" % DEFAULT_ULIMIT_NOFILE]
```

Reply:
The whole file is a mix of ' and " :) But you're right for the format string, it's cleaner. Fixing it

Reply:
I was talking about the format string indeed. The double quotes were just to match the other string on the same line :)